### DISCUSSION TITLE
Q: livestream.sh - log entries

### DISCUSSION THREAD

**Discussion by ghost** [mcguirepr89/BirdNET-Pi#791] [General]
I'm trying to dig into why my BirdNET-Pi stops working about once every day, but seems to resume after i open the webpage without clicking on any link (just displaying the self refreshing homepage).

When looking at logs, i'm regularily finding:
`Mar 19 23:05:38 birdy117 livestream.sh[3701178]: = 320.0kbits/s speed=   1x    #015size=  217945kB time=01:33:00.12 bitrate= 320.0kbits/s speed=   1x    #015size=  217965kB time=01:33:00.63 bitrate= 320.0kbits/s speed=   1x    #015size=  217985kB time=01:33:01.13 bitrate= 320.0kbits/s speed=   1x    #015size=  218004kB time=01:33:01.63 bitrate= 320.0kbits/s speed=   1x    #015size=  218024kB time=01:33:02.14 bitrate= 320.0kbits/s speed=   1x    #015size=  218043kB time=01:33:02.62 bitrate= 320.0kbits/s speed=   1x    #015size=  218063kB time=01:33:03.15 bitrate= 320.0kbits/s speed=   1x    #015size=  218082kB time=01:33:03.63 bitrate= 320.0kbits/s speed=   1x    #015size=  218102kB time=01:33:04.13 bitrate= 320.0kbits/s speed=   1x    #015size=  218122kB time=01:33:04.63 bitrate= 320.0kbits/s speed=   1x    #015size=  218141kB time=01:33:05.14 bitrate= 320.0kbits/s speed=   1x    #015size=  218161kB time=01:33:05.64 bitrate= 320.0kbits/s speed=   1x    #015size=  218181kB time=01:33:06.15 bitrate= 320.0kbits/s speed=   1x    #015size=  218200kB time=01:33:06.65 bitrate= 320.0kbits/s speed=   1x    #015size=  218221kB time=01:33:07.18 bitrate= 320.0kbits/s speed=   1x    #015size=  218240kB time=01:33:07.66 bitrate= 320.0kbits/s speed=   1x    #015size=  218259kB time=01:33:08.16 bitrate= 320.0kbits/s speed=   1x    #015size=  218280kB time=01:33:08.69 bitrate= 320.0kbits/s speed=   1x    #015size=  218299kB time=01:33:09.17 bitrate= 320.0kbits/s speed=   1x    #015size=  218318kB time=01:33:09.68 bitrate= 320.0kbits/s speed=   1x    #015size=  218339kB time=01:33:10.20 bitrate= 320.0kbits/s speed=   1x    #015size=  218358kB time=01:33:10.68 bitrate= 320.0kbits/s speed=   1x    #015size=  218378kB time=01:33:11.19 bitrate= 320.0kbits/s speed=   1x    #015size=  218397kB time=01:33:11.69 bitrate= 320.0kbits/s speed=   1x    #015size=  218417kB time=01:33:12.20 bitrate= 320.0kbits/s speed=   1x    #015size=  218436kB time=01:33:12.68 bitrate= 320.0kbits/s speed=   1x    #015size=  218456kB time=01:33:13.20 bitrate= 320.0kbits/s speed=   1x    #015size=  218476kB time=01:33:13.71 bitrate= 320.0kbits/s speed=   1x    #015size=  218496kB time=01:33:14.21 bitrate= 320.0kbits/s speed=   1x    #015size=  218515kB time=01:33:14.72 bitrate= 320.0kbits/s speed=   1x    #015size=  218535kB time=01:33:15.22 bitrate= 320.0kbits/s speed=   1x    #015size=  218555kB time=01:33:15.72 bitrate= 320.0kbits/s speed=   1x    #015size=  218574kB time=01:33:16.23 bitrate= 320.0kbits/s speed=   1x    #015size=  218594kB time=01:33:16.73 bitrate= 320.0kbits/s speed=   1x    #015size=  218614kB time=01:33:17.24 bitrate= 320.0kbits/s speed=   1x    #015size=  218633kB time=01:33:17.74 bitrate= 320.0kbits/s speed=   1x    #015size=  218653kB time=01:33:18.24 bitrate= 320.0kbits/s speed=   1x    #015size=  218673kB time=01:33:18.75 bitrate= 320.0kbits/s speed=   1x    #015size=  218693kB time=01:33:19.25 bitrate= 320.0kbits/s speed=   1x    #015size=  218712kB time=01:33:19.76 bitrate= 320.0kbits/s speed=   1x    #015size=  218732kB time=01:33:20.26 bitrate= 320.0kbits/s speed=   1x    #015size=  218752kB time=01:33:20.77 bitrate= 320.0kbits/s speed=   1x    #015size=  218771kB time=01:33:21.27 bitrate= 320.0kbits/s speed=   1x    #015size=  218790kB time=01:33:21.75 bitrate= 320.0kbits/s speed=   1x    #015size=  218811kB time=01:33:22.28 bitrate= 320.0kbits/s speed=   1x    #015size=  218830kB time=01:33:22.78 bitrate= 320.0kbits/s speed=   1x    #015size=  218850kB time=01:33:23.29 bitrate= 320.0kbits/s speed=   1x    #015size=  218870kB time=01:33:23.79 bitrate= 320.0kbits/s speed=   1x    #015size=  218888kB time=01:33:24.27 bitrate= 320.0kbits/s speed=   1x    #015size=  218908kB time=01:33:24.77 bitrate= 320.0kbits/s speed=   1x    #015size=  218928kB time=01:33:25.28 bitrate= 320.0kbits/s speed=   1x    #015size=  218948kB time=01:33:25.78 bitrate= 320.0kbits/s speed=   1x    #015size=  218967kB time=01:33:26.29 bitrate= 320.0kbits/s speed=   1x    #015size=  218987kB time=01:33:26.79 bitrate= 320.0kbits/s speed=   1x    #015size=  219007kB time=01:33:27.29 bitrate= 320.0kbits/s speed=   1x    #015size=  219026kB time=01:33:27.80 bitrate= 320.0kbits/s speed=   1x    #015size=  219046kB time=01:33:28.30 bitrate= 320.0kbits/s speed=   1x    #015size=  219066kB time=01:33:28.81 bitrate= 320.0kbits/s speed=   1x    #015size=  219085kB time=01:33:29.31 bitrate= 320.0kbits/s speed=   1x    #015size=  219105kB time=01:33:29.81 bitrate= 320.0kbits/s speed=   1x    #015size=  219125kB time=01:33:30.32 bitrate= 320.0kbits/s speed=   1x    #015size=  219144kB time=01:33:30.82 bitrate= 320.0kbits/s speed=   1x    #015size=  219164kB time=01:33:31.33 bitrate= 320.0kbits/s speed=   1x    #015size=  219184kB time=01:33:31.83 bitrate= 320.0kbits/s speed=   1x    #015size=  219203kB time=01:33:32.34 bitrate= 320.0kbits/s speed=   1x    #015size=  219223kB time=01:33:32.84 bitrate= 320.0kbits/s speed=   1x    #015size=  219242kB time=01:33:33.32 bitrate= 320.0kbits/s speed=   1x    #015size=  219263kB time=01:33:33.85 bitrate= 320.0kbits/s speed=   1x    #015size=  219281kB time=01:33:34.33 bitrate= 320.0kbits/s speed=   1x    #015size=  219302kB time=01:33:34.86 bitrate= 320.0kbits/s speed=   1x    #015size=  219321kB time=01:33:35.34 bitrate= 320.0kbits/s speed=   1x    #015size=  219340kB time=01:33:35.84 bitrate= 320.0kbits/s speed=   1x    #015size=  219361kB time=01:33:36.37 bitrate= 320.0kbits/s speed=   1x    #015size=  219381kB time=01:33:36.87 bitrate= 320.0kbits/s speed=   1x    #015size=  219400kB time=01:33:37.38 bitrate= 320.0kbits/s speed=   1x    #015size=  219420kB time=01:33:37.88 bitrate= 320.0kbits/s speed=   1x    #015size=  219440kB time=01:33:38.38 bitrate= 320.0kbits/s speed=   1x    #015size=  219459kB time=01:33:38.89 bitrate= 320.0kbits/s speed=   1x    #015size=  219479kB time=01:33:39.39 bitrate= 320.0kbits/s speed=   1x    #015size=  219499kB time=01:33:39.90 bitrate= 320.0kbits/s speed=   1x    #015size=  219518kB time=01:33:40.40 bitrate= 320.0kbits/s speed=   1x    #015size=  219538kB time=01:33:40.91 bitrate= 320.0kbits/s speed=   1x    #015size=  219558kB time=01:33:41.41 bitrate= 320.0kbits/s speed=   1x    #015size=  219578kB time=01:33:41.91 bitrate= 320.0kbits/s speed=   1x    #015size=  219597kB time=01:33:42.42 bitrate= 320.0kbits/s speed=   1x    #015size=  219617kB time=01:33:42.92 bitrate= 320.0kbits/s speed=   1x    #015size=  219637kB time=01:33:43.43 bitrate= 320.0kbits/s speed=   1x    #015size=  219657kB time=01:33:43.95 bitrate= 320.0kbits/s speed=   1x    #015size=  219677kB time=01:33:44.46 bitrate= 320.0kbits/s speed=   1x    #015size=  219696kB time=01:33:44.94 bitrate= 320.0kbits/s speed=   1x    #015size=  219715kB time=01:33:45.44 bitrate= 320.0kbits/s speed=   1x    #015size=  219735kB time=01:33:45.95 bitrate= 320.0kbits/s speed=   1x    #015size=  219755kB time=01:33:46.45 bitrate= 320.0kbits/s speed=   1x    #015size=  219774kB time=01:33:46.95 bitrate= 320.0kbits/s speed=   1x    #015size=  219794kB time=01:33:47.46 bitrate= 320.0kbits/s speed=   1x    #015size=  219813kB time=01:33:47.94 bitrate= 320.0kbits/s speed=   1x    #015size=  219833kB time=01:33:48.47 bitrate= 320.0kbits/s speed=   1x    #015size=  219853kB time=01:33:48.97 bitrate= 320.0kbits/s speed=   1x    #015size=  219872kB time=01:33:49.45 bitrate= 320.0kbits/s speed=   1x    #015size=  219893kB time=01:33:49.98 bitrate= 320.0kbits/s speed=   1x    #015size=  219912kB time=01:33:50.48 bitrate= 320.0kbits/s speed=   1x    #015size=  219932kB time=01:33:50.99 bitrate= 320.0kbits/s speed=   1x    #015size=  219952kB time=01:33:51.49 bitrate= 320.0kbits/s speed=   1x    #015size=  219970kB time=01:33:51.97 bitrate= 320.0kbits/s speed=   1x    #015size=  219991kB time=01:33:52.50 bitrate= 320.0kbits/s speed=   1x    #015size=  220011kB time=01:33:53.00 bitrate= 320.0kbits/s speed=   1x    #015size=  220029kB time=01:33:53.48 bitrate= 320.0kbits/s speed=   1x    #015size=  220050kB time=01:33:54.01 bitrate= 320.0kbits/s speed=   1x    #015size=  220070kB time=01`
(that's the full line - that's where it cuts off)

Is it correct to assume, that `livestream.sh` does have something to do with live audio function of the webpage? At the time, when i'm seeing the entry, no live audio is being streamed (by me) - no instance of the webpage open with live stream controls active even.

Can anyone briefly hint, what those log entries are about.

**Comment by ehpersonal38** [comment]
Yes so `livestream.sh` is exactly what you said, it's the live stream audio function that can either be accessed at the top right of the web interface, or by navigating to the "Spectrogram" page. The live audio stream is always running regardless of whether or not the web interface is there to play it, so that's normal behavior there. 

Though I'm a bit stumped as to why reloading the web interface would have any effect on the backend working. That's definitely not normal behavior! I kind of draw a similarity to [an issue](https://github.com/mcguirepr89/BirdNET-Pi/discussions/500#discussioncomment-3413436) a few users had a while back where their BirdNET-Pi would stop working until they SSH'd back into their Pi. The solution we arrived at was having them run this command on their Pi: `sudo systemctl enable getty@tty1.service`.

So, if you could try that and see if it makes a difference at all?

**Reply by ghost** [reply]
Hm. I have not found out why reloading the page got it working again. I wasn't able to link log entries to that action yet.
What i can say at least is, since i disabled livestream service totally I didn't have any outage yet. Still i will try the getty solution you quoted.
Will keep you posted.
